handbook/business-operations/security-audits.md

```diff
@@ -2,7 +2,32 @@
 This page contains explanations of the latest external security audits performed on Fleet software.
 
 ## June 2023 penetration testing of Fleet 4.33 
-Available on request.
+In June 2023, [Latacora](https://www.latacora.com/) performed an application penetration assessment of the application from Fleet.
+
+An application penetration test captures a point-in-time assessment of vulnerabilities, misconfigurations, and gaps in applications that could allow an attacker to compromise the security, availability, processing integrity, confidentiality, and privacy (SAPCP) of sensitive data and application resources. An application penetration
+
+test simulates the capabilities of a real adversary, but accelerates testing by using information provided by the target company.
+
+You can find the full report here: [2023-06-09-fleet-penetration-test.pdf](https://github.com/fleetdm/fleet/raw/main/docs/files/2023-06-09-fleet-penetration-test.pdf).
+
+### Findings
+#### 1 - Stored Cross-Site Scripting (XSS) in Tooltip
```

Comment:
~One of these headings probably isn't needed. Every title that we create we have to maintain forever.  I would say something like "See findings below:" at the end of the paragraph above.~

I say all that and then I see that you were just pattern matching the rest of what is here. Stick with what you have but....food for thought.